packages/perseus/src/strings.ts

```diff
@@ -675,133 +673,434 @@ export const strings = {
+            "Screen reader description for the number of points in the Polygon in the interactive graph widget.",
+        message: "The polygon has %(num)s points.",
+    },
+    srPolygonGraphPointsOne: {
+        context:
+            "Screen reader description for the number of points in the Polygon in the interactive graph widget when there is only one point.",
+        message: "The polygon has 1 point.",
+    },
+    srPolygonElementsNum: {
+        context:
+            "Screen reader description for the Polygon in the interactive graph widget explaining that it has a certain number of points.",
+        message: "A polygon with %(num)s points.",
+    },
+    srPolygonElementsOne: {
+        context:
+            "Screen reader description for the Polygon in the interactive graph widget explaining that it has one point.",
+        message: "A polygon with 1 point.",
+    },
+    srPolygonPointAngleApprox: {
+        context:
+            "Screen reader description for the angle of a point in the Polygon in the interactive graph widget when it's not an exact integer.",
+        message: "Angle approximately equal to %(angle)s degrees.",
+    },
+    srPolygonPointAngle: {
+        context:
+            "Screen reader description for the angle of a point in the Polygon in the interactive graph widget when it's an integer.",
+        message: "Angle equal to %(angle)s degrees.",
+    },
+    srPolygonSideLength: {
+        context:
+            "Screen reader description for the side of the Polygon in the interactive graph widget when its length is an exact integer.",
+        message:
+            "A line segment, length equal to %(length)s units, connects to point %(pointNum)s.",
+    },
+    srPolygonSideLengthApprox: {
+        context:
+            "Screen reader description for the side of the Polygon in the interactive graph widget when its length is not an exact integer.",
+        message:
+            "A line segment, length approximately equal to %(length)s units, connects to point %(pointNum)s.",
+    },
+    srUnlimitedPolygonEmpty: {
+        context:
+            "Screen reader description for the empty container that will eventually contain a Polygon in the interactive graph widget after the user has added points.",
+        message: "An empty coordinate plane.",
+    },
+    srSinusoidGraph: {
+        context:
+            "Aria label for the container containing a Sinusoid function in the interactive graph widget.",
+        message: "A sinusoid function on a coordinate plane.",
+    },
+    srSinusoidRootPoint: {
+        context:
+            "Aria label for the Point defining the midline intersection of the Sinusoid function in the interactive graph widget.",
+        message: "Midline intersection at %(x)s comma %(y)s.",
+    },
+    srSinusoidMaxPoint: {
+        context:
+            "Aria label for the Point defining max/min of the Sinusoid function in the interactive graph widget when it's above 0.",
```

Comment:
```suggestion
            "Aria label for the Point defining the maximum of the Sinusoid function in the interactive graph widget.",
```

I don't think the maximum being > 0 is a condition for this message to be shown, right? It's whenever the point is a local maximum (rather than a local minimum, or neither a min nor a max).